docs/data-sources/cce_nodes.md

```diff
@@ -33,6 +33,11 @@ The following arguments are supported:
 
 * `status` - (Optional, String) Specifies the status of the node.
 
+* `ignore_details` - (Optional, String) Specifies whether to get the details information of the nodes.
+  You can use this parameter to ignore some information you don't care about and make the query faster.
```

Comment:
it's better to modify the description of this parameter.

Reply:
fixed